src/yukkuri_game/game/yukkuri_components.py

```diff
@@ -158,24 +64,121 @@ class YukkuriStats:
     quality_score: float = 0.0
     discipline: float = 0.0
 
+@dataclass
+class Headline:
```

Comment:
Nit: Rename Headline to 'MemoryHeadline' or 'MemorySummary', currently not descriptive enough...